pootle/apps/pootle_fs/resources.py

```diff
@@ -47,3 +53,96 @@ def unsynced(self):
     @property
     def trackable_stores(self):
         return self.stores.exclude(obsolete=True).filter(fs__isnull=True)
+
+
+class FSProjectStateResources(object):
+
+    def __init__(self, context, pootle_path=None, fs_path=None):
+        self.context = context
+        self.pootle_path = pootle_path
+        self.fs_path = fs_path
+
+    def match_fs_path(self, path):
+        if not self.fs_path or fnmatch(path, self.fs_path):
+            return path
+
+    def _exclude_staged(self, qs):
+        return (
+            qs.exclude(staged_for_removal=True)
+              .exclude(staged_for_merge=True))
+
+    @cached_property
+    def fs_translations(self):
```

Comment:
hmm, yep - this defo needs to be renamed

its paths to found translation files